app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) < 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format. Use"
+                         " 'mv source_path destination_path'.")
+
+    source_path, destination_path = parts[1], parts[2]
+
+    if not os.path.isfile(source_path):
+        raise FileNotFoundError(f"Source file '{source_path}' "
+                                f"does not exist.")
+    split_file_extension = os.path.splitext(destination_path)
+    if not split_file_extension[-1]:
+        destination_path = os.path.join(destination_path, source_path)
```

Comment:
The current logic checks if the destination path has an extension to decide if it's a directory. However, this doesn't correctly handle cases where the destination path ends with a slash. You should check if the destination path ends with a slash and treat it as a directory in such cases.